charts/flink-operator/templates/misc.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-ingress-to-operator
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+spec:
+  podSelector:
+    matchLabels:
+      control-plane: controller-manager
+  policyTypes: [Ingress]
+  ingress:
+    - ports:
+      - port: 9443
+        protocol: TCP
+---
+# Grant the Flink Operator carte-blanche to connect to and manage
+# JobManager pods throughout the cluster.
+{{ if .Values.enableGlobalNetworkPolicy }}
```

Comment:
so I think we can get rid of this now, because the flink operator should be running in the same namespace as the flink job is it managing?